### DISCUSSION TITLE
command-line executable

### DISCUSSION THREAD

**Discussion by CamiloMM** [speechbrain/speechbrain#2367] [Feature Request]
### 🚀 The feature

I would love to just call the features in this toolkit from bash scripts and the like.

Is that a planned thing, or maybe there is one and I don't know about it? Maybe a recommended third party package?

### Solution outline

Similar to how OpenAI's Whisper has a command-line script which exposes the functionality to outside of Python scripts.

**Comment by Adel-Moumen** [comment]
Hey @mravanelli @TParcollet @pplantinga could you please have a look? Thanks.
